.github/workflows/build.yml

```diff
@@ -69,34 +69,6 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-  linux-ubuntu14:
```

Comment:
We should increase the version to `version: m105-adda216f-4` (look for `version: m105-adda216f-3` in this file)